### DISCUSSION TITLE
Make Full Width Image when parent has mx-3

### DISCUSSION THREAD

**Discussion by eryawanir** [tailwindlabs/tailwindcss#13032] [Help]
How to make this https://css-tricks.com/full-width-containers-limited-width-parents/ in tailwind?

**Comment by alex-krasikau** [comment]
Hey @eryawanir!

One of the options is using `w-screen ml-[calc(50%-50vw)]` on a full-width container. Here's a full example - https://play.tailwindcss.com/vrMgZD3T5Q

**Comment by AbhijitKhatua** [comment]
hi @eryawanir 
https://play.tailwindcss.com/RVSNGN2ein
